### ISSUE TITLE
Small values are exported in scientific format to pdxscene but can not be read back

### ISSUE THREAD

**Issue by wanderwurst** [SiliconStudio/paradox#306]
I have a component with an exposed field which is set to 0.0001

After saving the project in Paradox Studio (1.3ß) I can not re-open it, it fails with this error:

```
Reason: (Lin: 178, Col: 31, Chr: 8298) - (Lin: 178, Col: 38, Chr: 8305): Error while deserializing node [Scalar [anchor = , tag = , value = 1E-05.0, style = Plain, isPlainImplicit = True, isQuotedImplicit = False]] Exception: (Lin: 178, Col: 31, Chr: 8298) - (Lin: 178, Col: 38, Chr: 8305): Error while deserializing node [Scalar [anchor = , tag = , value = 1E-05.0, style = Plain, isPlainImplicit = True, isQuotedImplicit = False]]
```

The line in my MainScene.pdxscene reads

                        _linI: 1E-05.0

If I change that with a text editor to 0.00001 I can open the project, but when I save it (in Paradox Studio) the exponential formatting will be used again.

Maybe it has to do with my locale settings, I am on a german windows where all the number and date formattings are at their defaults (I can't find any system settings there though that would seem to have an effect on exponential formatting).

**Comment by xoofx** [comment]
Thanks for the report, this was indeed a bug in the Yaml serializer.

As a workaround, you can grab the latest version of SharpYaml from nuget (1.5.1) and drop the SharpYaml.dll into the `Bin\Windows-Direct3D11` folder of the Paradox distrib.

Our repository should also be updated by tomorrow in [`sources\common\deps\SharpYaml`](https://github.com/SiliconStudio/ssnet-deps/tree/master/SharpYaml)

We will release an official patch version in the coming days.

**Comment by xoofx** [comment]
This should be fixed by the new release 1.3.1

**Comment by wanderwurst** [comment]
Thank you very much! I keep being impressed by your incredibly fast response times. Highly appreciated! :-)
